wftests/warrior_tests/testcases/framework_tests/tc_kw_arg_type.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" ?>
+<Testcase>
+	<Details>
+		<Name>tc_kw_arg_type</Name>
+		<Title>Check keyword argument types</Title>
+		<Engineer>Arvind Kumar Sethuraman</Engineer>
+		<Date>2017-07-06</Date>
+		<Time>15:53</Time>
+		<State/>
+		<InputDataFile>No_Data</InputDataFile>
+		<Datatype/>
+		<default_onError action="next"/>
+		<Logsdir/>
+		<Resultsdir/>
+		<ExpectedResults/>
+	</Details>
+	<Requirements>
+		<Requirement/>
+	</Requirements>
+	<Steps>
+		<step Driver="ci_regression_driver" Keyword="check_kw_arg_type" TS="1">
```

Comment:
The test for this feature should actually compare the content of the file object provided in argument
with the actual file content
the actual file content can be provided as a string path in argument and read the file in a keyword

Reply:
moved these changes to war-991, along with addressing the review comments. The tests are also moved. Please let me know, if I should close this review request or otherwise.